conductr_cli/bndl_create.py

```diff
@@ -104,25 +108,27 @@ def bndl_create(args):
                 return 2
 
             process_oci = True
-        elif args.format == 'bundle':
-            peek = buff_in.peek(BNDL_PEEK_SIZE) if args.source is None else None
+        elif args.format == 'bundle' or args.format == 'configuration':
```

Comment:
We have to change https://github.com/typesafehub/conductr-cli/blob/master/conductr_cli/conduct_load.py#L251 so that `conduct load some-bundle some-bundle.conf` continues to work now that configuration is its own format.